modules/MessageHandler.js

```diff
@@ -3,6 +3,8 @@ const config = require('./Config');
 
 
 class MessageHandler {
```

Comment:
Remove this class since it will not be used anymore. Out Kademlia custom plugin will be used for messaging from now on.